src/main/resources/languages/lexvo-iso639-3.tsv

```diff
@@ -0,0 +1,7772 @@
+aaa	http://lexvo.org/id/iso639-3/aaa
```

Comment:
I guess this can be replaced by the java equivalent of `curl -I -L -H "Accept: application/rdf+xml"  http://lexvo.org/id/iso639-3/{code}`

Reply:
It is a good idea. I suggest we create a task for i because it requires a few changes. 

1) Create a field for the HttpClient (for mocking)
2) Make the class non final and the methods non static
3) Provide an instance of the LanguageMapper to all classes that need it.
4) Test the new method. 
5) I am not sure if we need to implement the reverse mapping as well (URI to ISO code)